src/utils/date.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Converts a date string to a timestamp.
+ * @param { Date } dateField - The date string in the format "YYYY-MM-DD" or a full ISO
+ * datetime string.
+ * @return { number } A number representing the timestamp.
+ */
+export const convertDateToTimestamp = (dateField: Date): number => {
```

Comment:
Based on the above we should allow the string type here. Since we can pass `string`, `number`, or `Date` object to `new Date()` constructor, then I think we can allow all these types in our function.

```suggestion
export const convertDateToTimestamp = (dateField: number | string | Date): number => {
```

Reply:
Addressed in https://github.com/threshold-network/website/commit/0f0f64c28a4043752e800d86acdacc80abb9fa74